packages/webpack-cli/__tests__/init/auto/init-auto.test.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable node/no-unpublished-require */
+'use strict';
+
+const firstPrompt = 'Will your application have multiple bundles?';
+const fs = require('fs');
+const { join } = require('path');
+const { run } = require('../../../../../test/utils/test-utils');
```

Comment:
Won't it make more sense to add these directly in `test` dir?

Reply:
Having them here makes more sense as they are integration tests with `webpack-cli`, at root we should have integration tests for `webpack` as suggested by @evilebottnawi above.

I think we should re structure `test-utils.js`, I would suggest to have it it's package of it own.

Reply:
Sounds good.

Reply:
But in a way root tests also use webpack-cli, and this is like testing the whole command from start to end so still makes more sense at root to me, @evilebottnawi what was your motivation?

Reply:
Yes indeed, having integration tests for `webpack-cli` at the root would be better. Infact this is the case for other packages.